README.md

```diff
@@ -23,12 +24,55 @@ yarn add @stoplight/json-schema-viewer
 
 ### Usage
 
-```ts
-import { Library } from "@stoplight/json-schema-viewer";
+```js
+export const theme = {
```

Comment:
Should we just export a reasonable dark/light theme from this project so that end user doesn't have to define it themselves?